plugins/obs-webrtc/whip-output.cpp

```diff
@@ -21,6 +21,9 @@ const uint8_t audio_payload_type = 111;
 const char *video_mid = "1";
 const uint8_t video_payload_type = 96;
 
+// Convert Kbps to 'How many 1500 Byte packets in 5 seconds'
+constexpr float videoNackStorageFactor = 125.0f / 1500.0f * 5.0f;
```

Comment:
This uses an MTU of 1500 bytes, but we're using a video fragment size of 1200 bytes above. Should this be adjusted accordingly, or in some way refer to that value?